fe/fe-core/src/test/java/com/starrocks/analysis/ShowRoutineLoadStmtTest.java

```diff
@@ -0,0 +1,55 @@
+// This file is made available under Elastic License 2.0.
```

Comment:
new files should use starrocks license header
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.